src/Form/matching.js

```diff
@@ -759,7 +734,9 @@ function getInputSubtype (input) {
  * @return {string}
  */
 const removeExcessWhitespace = (string = '') => {
-    return (string || '')
+    if (!string) return ''
+
+    return (string)
         .replace(/\n/g, ' ')
         .replace(/\s{2,}/g, ' ').trim()
```

Comment:
Since we call this a lot of times and pretty often with `null`, I'm short circuiting early instead of running two replaces and a trim on an empty string every time.